packages/manager/modules/pci/src/projects/project/instances/instances.html

```diff
@@ -184,6 +184,22 @@
                 data-ng-if="$row.statusGroup === 'ACTIVE' || $row.status === 'RESCUE'"
             >
             </oui-action-menu-divider>
+            <oui-action-menu-item
+                data-on-click="$ctrl.startInstance($row)"
+                data-disabled="$row.status === 'ACTIVE'"
```

Comment:
Could be great to add `isStarted()`  &  `isStopped()` (or `canStart()/canStop()` if needed) method in `Instance` class to avoid to check status in html.
Like https://github.com/ovh/manager/blob/master/packages/manager/modules/pci/src/components/project/instance/instance.class.js#L78

Reply:
yes right. i add them.